File: SDKs/googleapis.com/containeranalysis/v1alpha1/typescript/src/sdk/models/shared/googledevtoolscontaineranalysisv1alpha1gitsourcecontext.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



// GoogleDevtoolsContaineranalysisV1alpha1GitSourceContext
/** 
 * A GitSourceContext denotes a particular revision in a third party Git repository (e.g., GitHub).
**/
export class GoogleDevtoolsContaineranalysisV1alpha1GitSourceContext extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=revisionId" })
  revisionId?: string;

  @SpeakeasyMetadata({ data: "json, name=url" })
  url?: string;
}

File: SDKs/googleapis.com/containeranalysis/v1alpha1/go/pkg/models/shared/googledevtoolscontaineranalysisv1alpha1gitsourcecontext.go
package shared

// GoogleDevtoolsContaineranalysisV1alpha1GitSourceContext
// A GitSourceContext denotes a particular revision in a third party Git repository (e.g., GitHub).
type GoogleDevtoolsContaineranalysisV1alpha1GitSourceContext struct {
	RevisionID *string `json:"revisionId,omitempty"`
	URL        *string `json:"url,omitempty"`
}
